config/config.go

```diff
@@ -647,7 +649,9 @@ func init() {
 // execution can be triggered in a controlled way.
 // It's supposed to be called after init().
 // The configDirPath argument can be given to override the DefaultConfigDirPath.
-func Initialize(fs afero.Fs, configDirPath string) error {
+func initialize(fs afero.Fs, configDirPath string, loggerToUse io.Writer) error {
```

Comment:
I didn't want to change the signature of `Initialize`, so I've added a `InitalizeWithLogger` function